iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ShuntCompensator.java

```diff
@@ -45,21 +45,52 @@
     ShuntCompensator setCurrentSectionCount(int currentSectionCount);
 
     /**
-     * Get the susceptance for the current section count.
+     * Get the susceptance (in S) for the current section count i.e. the sum of the sections' susceptances for all sections in service.
+     * @see #getCurrentSectionCount()
      * <p>
      * Depends on the working variant.
      * @see VariantManager
      */
     double getCurrentB();
 
     /**
-     * Get the conductance for the current section count.
+     * Get the conductance (in S) for the current section count i.e. the sum of the sections' conductances for all sections in service.
+     * If a conductance of a section in service is undefined, the current conductance is undefined.
+     * @see #getCurrentSectionCount()
      * <p>
      * Depends on the working variant.
      * @see VariantManager
      */
     double getCurrentG();
 
+    /**
+     * Get the maximum susceptance (in S) i.e. the maximum of all the possible current susceptances.
```

Comment:
Looking to the documentation, it seems this method does a `max(Bi)` with i in [0, sectionCount[